test/groovy/com/sap/piper/UtilsTest.groovy

```diff
@@ -3,20 +3,36 @@ package com.sap.piper
 import org.junit.Rule
 import org.junit.Before
 import org.junit.Test
+import static org.junit.Assert.assertThat
 import org.junit.rules.ExpectedException
+import org.junit.rules.RuleChain
 
-import com.sap.piper.Utils
+import static org.hamcrest.Matchers.hasItem
+import static org.hamcrest.Matchers.is
+import static org.hamcrest.Matchers.containsString
+
+import util.JenkinsLoggingRule
+import util.JenkinsShellCallRule
+import util.BasePiperTest
+import util.Rules
 
+import com.sap.piper.Utils
 
-class UtilsTest {
+class UtilsTest extends BasePiperTest {
+    private ExpectedException thrown = ExpectedException.none()
+    private JenkinsLoggingRule jlr = new JenkinsLoggingRule(this)
+    private JenkinsShellCallRule jscr = new JenkinsShellCallRule(this)
 
     @Rule
-    public ExpectedException thrown = new ExpectedException().none()
+    public RuleChain rules = Rules
+        .getCommonRules(this)
+        .around(thrown)
+        .around(jscr)
+        .around(jlr)
 
-    private utils = new Utils()
+    //private utils = new Utils()
```

Comment:
IMO we should not keep this kind of commented coding. --> I propose to remove the line.